frontend/src/main/components/Commons/ManageCows.js

```diff
@@ -1,38 +1,60 @@
 import React from "react";
 import { Card, Button, Row, Col } from "react-bootstrap";
-import cowHead from "./../../../assets/CowHead.png"; 
+import cowHead from "./../../../assets/CowHead.png";
 
-// add parameters 
-const ManageCows = ({userCommons,commons, onBuy, onSell}) =>  {
-    // update cowPrice from fixture
+const ManageCows = ({ userCommons, commons, onBuy, onSell }) => {
     return (
         <Card>
-        <Card.Header as="h5">Manage Cows</Card.Header>
-        <Card.Body>
-            {/* change $10 to info from fixture */}
-            <Card.Title>Market Cow Price: ${commons?.cowPrice}</Card.Title>
-            <Card.Title>Number of Cows: {userCommons.numOfCows}</Card.Title>
-            <Card.Title>Current Milk Price: ${commons?.milkPrice} </Card.Title>
+            <Card.Header as="h5">Manage Cows</Card.Header>
+            <Card.Body>
+                <Card.Title>
+                    Market Cow Price: ${commons?.cowPrice?.toFixed(2)}
+                </Card.Title>
+                <Card.Title>Number of Cows: {userCommons.numOfCows}</Card.Title>
```

Comment:
newline here to match the others?

Reply:
We could, I was just following Prettier because its used in industry a lot.